public/js/index.js

```diff
@@ -1,5 +1,32 @@
-function displayMessage() {
-  return 'Hello, World!';
+function buildProduct(product) {
```

Comment:
Thank you for clarifying!

So if we want to read the code lineally... ¿Shouldn't we also place the entry point code to the top of the file?

1. `app.appendChild(buildProducts(products));`
2. `buildProduct(...) {}`
3. `buildProducts(...) {}`